app/ui-react/packages/ui/src/Data/Virtualizations/Views/ViewsImportLayout.css

```diff
@@ -0,0 +1,43 @@
+.views-import-layout {
+  height: calc(100% - 36px - 60px);
+}
+
+.views-import-layout__body {
+  max-height: none;
+}
+
+.views-import-layout.has-footer .views-import-layout__body {
+  height: calc(100vh - 60px - 36px - 57px - 61px);
+  max-height: none;
+}
+
+@media (min-width: 768px) {
+  .views-import-layout__body {
+    height: calc(100vh - 121px - 60px - 36px);
+    max-height: none;
+  }
+
+  .views-import-layout.has-footer .views-import-layout__body {
+    height: calc(100vh - 121px - 60px - 36px - 61px);
+    max-height: none;
+  }
+}
+
+.views-import-layout__contentWrapper {
+  overflow: auto;
+  padding: 0;
+}
+
+.views-import-layout__content {
+  max-width: 1100px;
+  margin: auto;
+}
+
+.views-import-layout__footer {
+  padding: 17px 15px;
+  z-index: 1;
+}
+
+.list-view-pf .list-group-item-heading {
```

Comment:
This will clash with other list views around the app, please namespace.

Reply:
amended the PR with modified stylesheets